apps/site/src/app/(home)/sections/Countdown/useWindow.tsx

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from "react";
+
+export default function useWindow() {
+	const [size, setSize] = useState([0, 0]);
+	useEffect(() => {
+		setSize([window.innerWidth, window.innerHeight]);
+		function setWindowSize() {
+			setSize([window.innerWidth, window.innerHeight]);
+		}
```

Comment:
nitpick: you could call `setWindowSize` after it has been defined instead of calling `setSize([window.innerWidth, window.innerHeight])` beforehand